media-gfx/blender/blender-4.4.3-r1.ebuild

```
@@ -0,0 +1,900 @@
+
+		# Rendering:
+		-DWITH_HYDRA="no" # TODO: Package Hydra
+
+		# Rendering (Cycles):
+		-DWITH_CYCLES_OSL="$(usex osl)"
+		-DWITH_CYCLES_EMBREE="$(usex embree)"
+		-DWITH_CYCLES_PATH_GUIDING="$(usex openpgl)"
+
+		-DWITH_CYCLES_DEVICE_OPTIX="$(usex optix)"
+		-DWITH_CYCLES_DEVICE_CUDA="$(usex cuda)"
+		-DWITH_CYCLES_CUDA_BINARIES="$(usex cuda "$(usex cycles-bin-kernels)")"
+
+		-DWITH_CYCLES_DEVICE_HIP="$(usex hip)"
+		-DWITH_CYCLES_HIP_BINARIES="$(usex hip "$(usex cycles-bin-kernels)")"
+		-DWITH_CYCLES_HYDRA_RENDER_DELEGATE="no" # TODO: package Hydra
+
+		# -DWITH_CYCLES_STANDALONE=OFF
+		# -DWITH_CYCLES_STANDALONE_GUI=OFF
+
+		-DWITH_BLENDER_THUMBNAILER="yes"
+	)
+
+	if has_version ">=dev-python/numpy-2"; then
+		mycmakeargs+=(
+			-DPYTHON_NUMPY_INCLUDE_DIRS="$(python_get_sitedir)/numpy/_core/include"
+			-DPYTHON_NUMPY_PATH="$(python_get_sitedir)/numpy/_core/include"
+		)
+	fi
+
+	# requires dev-vcs/git
+	if [[ "${PV}" == *9999* && "${BVC}" == "alpha" ]]; then
+		mycmakeargs+=(
+			# -DWITH_BUILDINFO="no"
+			-DWITH_EXPERIMENTAL_FEATURES="$(usex experimental)"
+		)
+	else
+		mycmakeargs+=(
+			-DWITH_EXPERIMENTAL_FEATURES="no"
+		)
+	fi
+
+	if use cuda; then
+		# Ease compiling with required gcc similar to cuda_sanitize but for cmake
+		if use cycles-bin-kernels; then
+			local -x CUDAHOSTCXX="$(cuda_gccdir)"
+			local -x CUDAHOSTLD="$(tc-getCXX)"
+
+			if [[ -n "${CUDAARCHS}" ]]; then
+				mycmakeargs+=(
+					-DCYCLES_CUDA_BINARIES_ARCH="$(echo "${CUDAARCHS}" | sed -e 's/^/sm_/g' -e 's/;/;sm_/g')"
+				)
+			fi
+		fi
+	fi
+
+	if use hip; then
+		mycmakeargs+=(
+			-DHIP_ROOT_DIR="$(hipconfig -p)"
+
+			-DHIP_HIPCC_FLAGS="-fcf-protection=none"
+
+			-DCMAKE_HIP_LINK_EXECUTABLE="$(get_llvm_prefix)/bin/clang++"
+
+			-DCYCLES_HIP_BINARIES_ARCH="$(get_amdgpu_flags)"
+		)
+	fi
+
+	if use hiprt; then
+		mycmakeargs+=(
+			-DHIPRT_ROOT_DIR="${ESYSROOT}/usr/lib/hiprt/2.5"
+		)
+	fi
+
+	if use optix; then
+		mycmakeargs+=(
+			-DCYCLES_RUNTIME_OPTIX_ROOT_DIR="${ESYSROOT}/opt/optix"
+			-DOPTIX_ROOT_DIR="${ESYSROOT}/opt/optix"
+		)
+	fi
+
+	if use wayland; then
+		mycmakeargs+=(
+			-DWITH_GHOST_WAYLAND_APP_ID="blender-${BV}"
+			-DWITH_GHOST_WAYLAND_LIBDECOR="$(usex gnome)"
+		)
+	fi
+
+	# This is currently needed on arm64 to get the NEON SIMD wrapper to compile the code successfully
+	use arm64 && append-flags -flax-vector-conversions
+
+	append-cflags "$(usex debug '-DDEBUG' '-DNDEBUG')"
+	append-cxxflags "$(usex debug '-DDEBUG' '-DNDEBUG')"
+
+	if tc-is-gcc; then
+		# We disable these to respect the user's choice of linker.
+		mycmakeargs+=(
+			-DWITH_LINKER_GOLD="no"
+		)
+	fi
+
+	if tc-is-clang || use osl; then
+		mycmakeargs+=(
+			-DWITH_CLANG="yes"
+			-DWITH_LLVM="yes"
+		)
+	fi
+
+	if use test; then
+		local CYCLES_TEST_DEVICES=( "CPU" )
+		if use cycles-bin-kernels; then
+			use cuda && CYCLES_TEST_DEVICES+=( "CUDA" )
+			use optix && CYCLES_TEST_DEVICES+=( "OPTIX" )
+			use hip && CYCLES_TEST_DEVICES+=( "HIP" )
```

```suggestion
			use hip && CYCLES_TEST_DEVICES+=( "HIP" )
			use hiprt && CYCLES_TEST_DEVICES+=( "HIP-RT" )
```

So we test rendering via hiprt.